src/test/run-make/nvptx-binary-crate/Makefile

```diff
@@ -0,0 +1,9 @@
+-include ../../run-make-fulldeps/tools.mk
+
+ifeq ($(TARGET),nvptx64-nvidia-cuda)
+all:
+	$(RUSTC) main.rs -Clink-arg=--arch=sm_60 --crate-type="bin" -O --target $(TARGET)
+	FileCheck main.rs --input-file $(TMPDIR)/main.ptx
```

Comment:
Hmm, it feels like codegen test infrastructure should be used. At very least this is trying to use the `FileCheck` in `$PATH`, which is not correct.

Reply:
Interesting, at first I was afraid it will use system-wide `FileCheck` but on my env, either `bootstrap` or `compiletest` add a correct `$PATH`. So, the tests pass no matter the system-wide binary exist or not.

Reply:
Okay then. Still feels suspect, but I’m fine with that explanation :)